internal/cmd/request.go

```diff
@@ -26,15 +26,13 @@ func MustHandleCMDRequestAndExit(endpoint string, request any, responseType any)
 func MustHandleCMDRequestAndExitFull(endpoint string, request any, responseType any, options CommonOptions, customPrintFunc CustomResponseFormatter) {
 	response, err := SendCMDRequest(endpoint, request)
 	if err != nil {
-		log.Fatal("Failed to send the CMD request.", err)
+		log.Fatal("Failed to send the CMD request.", err, log.NewAttr("endpoint", endpoint))
 	}
 
 	if !response.Success {
-		log.Error("API response was unsuccessful.", log.NewAttr("message", response.Message))
+		log.Fatal("API Request was unsuccessful.", log.NewAttr("message", response.Message))
 
-		exit.Exit(2)
-
-		// Return after setting the exit code to avoid overwriting it during tests.
+		// Return after log.Fatal sets the exit code to 1 to avoid overwriting the exit code during tests.
```

Comment:
Nit (I would normally not hold up an approval for this, but there is another change you need to make so I may as well include it).

We we mention functions, we usually add parens after (e.g. `log.Fatal()`).